angular.json

```diff
@@ -33,6 +33,22 @@
             "scripts": []
           },
           "configurations": {
+            "staging": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.stage.ts"
```

Comment:
Which environment does `staging` refer to?  We use both `hceutesting.com` and `hceu-performance.com` (a.k.a CI)

Reply:
Will remove the staging environment, and next sprint will work on moving the url setting from this component to the CEMS module.  See https://homeceu.atlassian.net/browse/CEMS-2162